app/src/main/java/io/github/androidacademyglobal/di/AppModule.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.androidacademyglobal.di
+
+import io.github.androidacademyglobal.profile.ProfileViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+val appModule = module {
+    viewModel { ProfileViewModel() }
```

Comment:
I'm not familiar with `Koin` yet but do you really need to manually create an object? or maybe some annotation could do it for you like Dagger/Hilt does (I also wonder - how do you create a complex object containing refs to other objects - e.g. `*UseCase` in `VM`)

Reply:
It's classic Koin usage like `single { MyDependency(get(), get(), get(), get()) }` They introduced [another method](https://insert-koin.io/docs/reference/koin-android/dsl-update) in 3.2

Reply:
Thanks, now I know smth about Koin :)